build.psm1

```diff
@@ -1251,46 +1298,68 @@ function Test-PSPesterResults
 
 
 function Start-PSxUnit {
-    [CmdletBinding()]param()
-
-    log "xUnit tests are currently disabled pending fixes due to API and AssemblyLoadContext changes - @andschwa"
-    return
-
-    if ($Environment.IsWindows) {
-        throw "xUnit tests are only currently supported on Linux / OS X"
-    }
-
-    if ($Environment.IsMacOS) {
-        log "Not yet supported on OS X, pretending they passed..."
-        return
-    }
+    [CmdletBinding()]param(
+        [string] $TestResultsFile = "XUnitResults.xml"
+    )
 
     # Add .NET CLI tools to PATH
     Find-Dotnet
 
-    $Arguments = "--configuration", "Linux", "-parallel", "none"
-    if ($PSCmdlet.MyInvocation.BoundParameters["Verbose"].IsPresent) {
-        $Arguments += "-verbose"
-    }
+    $options = New-PSOptions
 
     $Content = Split-Path -Parent (Get-PSOutput)
```

Comment:
can this be simplified to just:
`if ( -not (test-path (get-psoutput))) `

Reply:
We use `$Content` later. So I will keep it as is.